index.js

```diff
@@ -13,45 +15,76 @@ export default class AddonInstaller extends Plugin {
       'default',
       (args, res) => {
         const channelID = args[0].channel?.id;
-        const isPlugin = channelID === '753291447523868753' && true;
-
-        if (isPlugin || channelID === '753291485100769411') {
-          const addonURL = args[0].message?.embeds[0]?.fields[0].rawValue;
-          const addonID = addonURL.split('/').pop().toLowerCase();
-
-          const pluginsOrThemes = isPlugin ? 'plugins' : 'themes';
-
-          const addonIsInstalled = vizality.manager[
-            pluginsOrThemes
-          ].keys.includes(addonID);
-
-          res.props.children.push(
-            <>
-              <ContextMenu.Separator />
-              <ContextMenu.Group>
-                <ContextMenu.Item
-                  label={`${addonIsInstalled ? 'Uninstall' : 'Install'} ${
-                    isPlugin ? 'Plugin' : 'Theme'
-                  }`}
-                  id="addon-installer"
-                  action={async () => {
-                    if (addonIsInstalled) {
-                      await vizality.manager[pluginsOrThemes].uninstall(
-                        addonID
-                      );
-                    } else {
-                      await vizality.manager[pluginsOrThemes].install(addonURL);
-                    }
-                  }}
-                />
-              </ContextMenu.Group>
-            </>
-          );
-        }
+        if (channelID === '755005584322854972') {
+          if (vizality.manager.plugins.isInstalled('00pccompat') && vizality.manager.plugins.isEnabled('00pccompat')) {
+            const addonURL = args[0].message?.content.match(githubRegex)[0];
+            const addonID = addonURL.split('/').pop().toLowerCase();
+            const isPlugin = channelID === '755005584322854972' && true;
```

Comment:
Variables with the same name defined twice. Consider defining at a higher level and making it work for both scenarios.

Reply:
Additionally, the `&& true` is unnecessary. You can achieve the same result with `const isPlugin = channelID === '755005584322854972';`.

Reply:
In this specific scenario, `isPlugin` will always be true because it is inside an if statement checking that `channelID === '755005584322854972'` and is therefore entirely unnecessary.

Reply:
Done as per 4f1b7eb22b6c6933af4d221f82d970bc5a645c9d